src/helm/benchmark/slurm_runner.py

```diff
@@ -268,11 +269,20 @@ def _submit_slurm_job_for_run_spec(self, run_spec: RunSpec) -> int:
             "chdir": os.getcwd(),
         }
         # TODO: Move resource requirements into RunSpec.
+        slurm_node_names = os.getenv(_SLURM_NODE_NAMES_ENV_NAME)
         if run_spec.name.startswith("msmarco:"):
             raw_slurm_args["mem"] = "64G"
         if "device=cuda" in run_spec.name:
             raw_slurm_args["gres"] = "gpu:1"
             raw_slurm_args["partition"] = "jag-hi"
+        if "model=huggingface" in run_spec.name:
+            raw_slurm_args["gres"] = "gpu:1"
+            raw_slurm_args["partition"] = "sphinx"
+            if not slurm_node_names or "sphinx" not in slurm_node_names:
```

Comment:
Should we say that if this environment variable doesn't exist, it defaults to all sphinxes or is that dangerous?

Reply:
Defaulting to all sphinxes may lead to GPU fragmentation which is undesirable (e.g. if we use one GPU per sphinx, other Slurm users cannot run jobs that require 8 GPUs on a single machine). Also we should stay off sphinx3.